samples/dashboard/demos/component-html/demo.js

```diff
@@ -0,0 +1,45 @@
+const dashboard = new Dashboard.Dashboard('container', {
+    gui: {
+        enabled: true,
+        layouts: [{
+            rows: [{
+                cells: [{
+                    id: 'dashboard-1'
+                }, {
+                    id: 'dashboard-2'
+                }]
+            }, {
+                cells: [{
+                    id: 'dashboard-3'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        type: 'html',
+        cell: 'dashboard-1',
+        elements: [{
+            tagName: 'img',
+            attributes: {
+                src: 'https://www.highcharts.com/docs/assets/images/hollow-candlestick-89ede73673a823ad1e222f9280da27e8.png'
```

Comment:
I found it a bit confusing, that these are pictures of charts. Some more obvious HTML content would be nice. Maybe the Highsoft logo for the first one and this [embedded video](https://vimeo.com/643496165) for the second one?